pkg/chart/gcp.go

```diff
@@ -443,3 +443,55 @@ func roleBindingExistsInGCP(ctx context.Context, gcpProject, teamID, role string
 	roles := strings.Split(stdOut.String(), "\n")
 	return slices.Contains(roles, role), nil
 }
+
+func grantSATokenCreatorRole(ctx context.Context, teamID, gcpProject string) error {
+	role := "roles/iam.serviceAccountTokenCreator"
+
+	var sa = fmt.Sprintf("%v@%v.iam.gserviceaccount.com", teamID, gcpProject)
```

Comment:
I'd use := variable declaration, same as role above, but that's up to you.

Reply:
Ok!